src/hibp.js

```diff
@@ -50,6 +50,8 @@ const hibp = {
                 throw new Error(ERR403);
               case 404:
                 return null;
+              default:
+                throw new Error(err.response.data);
```

Comment:
`err.response.data` is likely to be empty for an unexpected error. The only time he uses this currently is for a 429 response, which we should handle explicitly as its own case. Let's use `err.response.statusText` (e.g. "Too Many Requests") instead.